elements/campaign-product-shot/campaign-product-shot.js

```diff
@@ -0,0 +1,46 @@
+import {
+  registerElement,
+  BulbsHTMLElement,
+} from 'bulbs-elements/register';
+import { filterBadResponse, getResponseJSON } from 'bulbs-elements/util';
+import invariant from 'invariant';
+
+class CampaignProductShot extends BulbsHTMLElement {
+
+  handleRequestSuccess (data) {
+    if (!data.product_shot_url) {
+      console.log('Product shot url not set on this campaign');
```

Comment:
@collin are we doing `console.log` calls in this code for errors like this?

If so, include some campaign data with this message for easier debugging (e.g. the id or name).